### PR TITLE
Update 'quick-start-static' and 'start-with-project-stub' for new bem.info

### PR DESCRIPTION
**Pull request by vithar**
//cc @tadatuta

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,24 @@
+libs/
+node_modules/
+.DS_Store
+.idea
+.project
+.svn
+npm-debug.log
+*.swp
+
+# bem
```
Comment (tadatuta) [inline .gitignore]: there's nothing like these in bem-method

File: articles/quick-start-static/quick-start-static.en.md [hunk 2]
```diff
@@ -1,79 +1,74 @@
...
+**Note** If your operating system is Windows, you must run the following commands in Git Bash with administrator rights.
 
 1.  Make a local copy of `project-stub`.
 
-    **NB** Do not use root rights to install npm and bower dependencies. bower dependencies are installed in the `libs` directory by npm postinstall.
+    **Note** Do not use root rights to install npm and bower dependencies. bower dependencies are installed in the `libs` directory by `npm postinstall`.
 
     ```bash
-    git clone https://github.com/bem/project-stub.git --depth 1 --branch v1.3.1 start-project
+    git clone https://github.com/bem/project-stub.git --depth 1 start-project
```
Comment (tadatuta) [inline articles/quick-start-static/quick-start-static.en.md]: it won't be reproducible without tag

File: articles/quick-start-static/quick-start-static.en.md [hunk 3]
```diff
@@ -1,79 +1,74 @@
...
 
-2.  Run the server using [ENB](https://ru.bem.info/tools/bem/enb-bem-techs/) (this article is available only in Russian).
+2.  Run the server using [ENB](https://en.bem.info/toolbox/enb/):
 
     ```bash
     npm start
     ```
 
-3.  Check the result on `http://localhost:8080/desktop.bundles/index/index.html`.
+3.  Check the result on [http://localhost:8080/desktop.bundles/index/index.html](http://localhost:8080/desktop.bundles/index/index.html).
```
Comment (tadatuta) [inline articles/quick-start-static/quick-start-static.en.md]: will result in false negative with broken links checker

File: articles/quick-start-static/quick-start-static.en.md [hunk 4]
```diff
@@ -1,79 +1,74 @@
...
+2.  [Create a block](#create-a-block)
+3.  [Implement the hello block](#implement-the-hello-block)
+  * [Use JavaScript technology](#implement-the-hello-block-in-javascript-technology)
+  * [Use BEMHTML technology](#implement-the-hello-block-in-bemhtml-technology)
+  * [Use CSS technology](#implement-the-hello-block-in-css-technology)
 
-When all steps have been completed you can watch the [result](#result).
+When all steps have been completed you can watch the [result](#the-final-result).
 
-<a name="page-creation"></a>
```
Comment (tadatuta) [inline articles/quick-start-static/quick-start-static.en.md]: changing anchors may result in broken links on other resources

File: articles/start-with-project-stub/start-with-project-stub.en.md [hunk 5]
```diff
@@ -537,251 +565,265 @@ An extra trick: if you would like to avoid cascade when styling the block, mark
...
+
+It provides just one block `box` that does all we need.
 
 You should declare a library name, its version (if available) and its repository URL in a `bower.json` file.
 
-```
-    "dependencies": {
-        "bem-components": "2.4.0",
-        "j": "git://github.com/innabelaya/j.git#695d479fbdd7c97e61bd89953ef095e2e567e70e"
+```json
```
Comment (tadatuta) [inline articles/start-with-project-stub/start-with-project-stub.en.md]: closing `}` is missed

File: articles/quick-start-static/quick-start-static.en.md (hunk 2 above)
Comment (vithar) [inline articles/quick-start-static/quick-start-static.en.md]: We need a check-list for project-stub version release in order to remember update these two documents.

File: articles/quick-start-static/quick-start-static.en.md (hunk 3 above)
Comment (vithar) [inline articles/quick-start-static/quick-start-static.en.md]: We can add localhost links to black list of broken links checker.

File: articles/quick-start-static/quick-start-static.en.md (hunk 4 above)
Comment (vithar) [inline articles/quick-start-static/quick-start-static.en.md]: I'll check our docs and I think to external links to these documents.

File: articles/start-with-project-stub/start-with-project-stub.en.md (hunk 5 above)
Comment (vithar) [inline articles/start-with-project-stub/start-with-project-stub.en.md]: Его и не было. Добавлю.

File: .gitignore (hunk 1 above)
Comment (vithar) [inline .gitignore]: I leave

```
.DS_Store
*.swp
```

**Pull request merged**